onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -93,6 +93,9 @@
 #include "orttraining/core/graph/optimizer_config.h"
 #include "orttraining/core/framework/mpi_setup.h"
 #include "orttraining/core/framework/data_transfer_utils.h"
+#else
+#include "core/session/IOBinding.h"
+#include "../orttraining/orttraining/core/framework/data_transfer_utils.h"
```

Comment:
>../orttraining [](start = 10, length = 14)

can skip this

Reply:
actually ../orttraining is not set in inference build. However I shall refactor data data_trainfer_utils.h if it is not only for training.

Reply:
it got moved out of orttraining, now it should be in onnxruntime/core/framework

---
In reply to: [394008206](https://github.com/microsoft/onnxruntime/pull/3235#discussion_r394008206) [](ancestors = 394008206)